src/adjusters.jl

```diff
@@ -18,7 +18,7 @@ Base.trunc(zdt::ZonedDateTime, ::Type{Millisecond}) = zdt
 # Adjusters
 for prefix in ("firstdayof", "lastdayof"), suffix in ("week", "month", "year", "quarter")
     func = Symbol(prefix * suffix)
```

Comment:
Not related but is it cleaner to do:

```suggestion
    func = Symbol(prefix, suffix)
```